sky/resources.py

```diff
@@ -841,6 +841,13 @@ def _try_validate_image_id(self) -> None:
                         f'larger than the specified disk_size: {self.disk_size}'
                         ' GB. Please specify a larger disk_size to use this '
                         'image.')
+            elif image_size == self.disk_size:
```

Comment:
Can we merge this with the previous `if` branch to reduce duplicated code? i.e. `if image_size >= self.disk_size:`

Reply:
Certainly, I have shortened the code in the latest commit :)